### DISCUSSION TITLE
How do you update the display options on your virtual display?

### DISCUSSION THREAD

**Discussion by flowerdealer** [VirtualDrivers/Virtual-Display-Driver#14] [Q&A]
So I defined the resolutions I wanted on my virtual display. I need to update them later, how do I add new ones to the options.txt file. Do I need to add and reinstall the driver?

**Comment by james-s-w-clark** [comment]
- add to options.txt
- reinstall driver
- in Windows display settings, select the virtual monitor:
![image](https://github.com/user-attachments/assets/a63b59d7-87ec-47b2-b583-24c5fe3dd388)
- then, edit the resolution and/or advanced info (for refresh rate):
![image](https://github.com/user-attachments/assets/adaa8ea5-2f9e-4317-941e-51e6e6d6c8f2)
